docs/data/sources/google-tag-manager-client-legacy.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Amplitude Analytics Legacy
+description: Official Google Tag Manager template for Amplitude Analytics for data collection.
```

Comment:
I would be more clear about which template this is in the description. This shows up in the site's meta in Google Search results, and shows up on all our social preview cards.

Reply:
Rewrite the description here and added the full name of GTM template.